tox.ini

```diff
@@ -22,4 +22,4 @@ commands =
     touch tests/__init__.py
     flake8 edx_prefectutils tests
     rm tests/__init__.py
-    isort --check-only --diff --recursive edx_prefectutils tests
```

Comment:
Do we not want to check the subdirectories(if any) ?

Reply:
So in isort latest version we don't need to explicitly put `--recursive` option. It something that's now a default behaviour 
Details are mentioned here : https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html
If we use this option we getting following warnings
`UserWarning: W0501: The following deprecated CLI flags were used and ignored: --recursive!`